### DISCUSSION TITLE
Static Site Generators

### DISCUSSION THREAD

**Discussion by rubenmarcus** [frontendbr/forum#451] [Perguntas e Respostas]
Oi pessoal,venho pensando em fazer meu portfolio ainda esse mês e vi que uma solução bem simples e rápida seria usar um Gerador de Site Estático.

Gostei muito desse aqui: https://nuxtjs.org/ Porque é feito em Vue. Alguem mais já usou ele?

Sabe se o lance das rotas, atrapalha em SEO ? procurei artigos sobre problemas de SEO no google com rotas dinamicas em vue e não vi nenhum. Mas lembro que no Angular isso era um problema constante, e uma das soluções seria usar o [pre-render](https://prerender.io/)

Valeu

**Comment by kelciocajueiro** [comment]
Dá uma olhada em https://surge.sh/

**Comment by rubenmarcus** [comment]
Legal, aqui https://nuxtjs.org/examples/nested-routes , Eu sou bem leigo em Vue, nunca coloquei em pratica, mas com o surge caso existisse algum problema de url dinamica por javascript ele cuidaria é isso?

**Comment by rubenmarcus** [comment]
Hoje no meu [Portfolio](http://rubenmarcus.com), eu já tenho alguns [jsons](https://github.com/rubenmarcus/old-portfolio/tree/master/data) que são consumidos pelo angular..
Queria tentar mante-los. Mas com um gerador estático talvez usando Vue

**Comment by expalmer** [comment]
Eu uso [metalsmith.io](http://metalsmith.io) no [Meu Blog](http://expalmer.github.io)

**Comment by advec** [comment]
Até mês passado era apaixonado pelo Jekyll, mas hoje o Hexo me ganhou! Meu maior motivo para gostar dele é por ser desenvolvido em javascript.

**Comment by hugocarreira** [comment]
Atualmente utilizo Jekyll no [meu blog](http://hugocarreira.me), sou meio suspeito para falar pois nunca experimentei outros generators, pois o Jekyll sempre me proporcionou uma boa experiência e ele me oferece tudo que eu preciso. 

Tenho dois motivos que fazem ele ser meu xodózinho: 

A curva de aprendizado dele é absurdamente baixa da para fazer muita coisa em pouco tempo; Tem o curso (gratuito) do @willianjusten que é **MUITO** bom, eu aprendi a maioria das coisas que sei sobre Jekyll fazendo o curso.

A integração dele com os apps de desenvolvimento é muito boa (Sublime / Atom), os snippets existentes são ótimos, consigo iniciar um post ou um draft com meu template default de cabeçalho com dois cliques. E publicar os drafts como post, também com dois cliques.
